src/common/util/include/openvino/util/filesystem.hpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/core/visibility.hpp"
+
+#if defined(_MSC_VER) && defined(CPP_VER_11)
+#    define OPENVINO_HAS_EXP_FILESYSTEM 1
```

Comment:
Is required to set value to FS macros?
Maybe we can check if is defined or not?

Reply:
Done